### PR TITLE
Init project

### PR DESCRIPTION
**Pull request by senyer**
- Integrated springboot

- Configure base html page

- Defining the basic components

### CODE REVIEW

**Review by ou-taku14 — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.springframework.boot</groupId>
+    <artifactId>spring-boot-starter-parent</artifactId>
+    <version>2.1.13.RELEASE</version>
+    <relativePath/> <!-- lookup parent from repository -->
+  </parent>
+  <groupId>com.cloud</groupId>
+  <artifactId>distkv-cloud</artifactId>
+  <version>0.0.1-SNAPSHOT</version>
```
Comment (jovany-wang) [inline pom.xml]: 0.1.0-SNAPSHOT

File: src/main/java/com/cloud/DistkvCloudApplication.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package com.cloud;
```
Comment (jovany-wang) [inline src/main/java/com/cloud/DistkvCloudApplication.java]: package cloud.distkv

File: src/main/java/com/cloud/common/annotations/DoAuth.java [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target({ElementType.METHOD, ElementType.TYPE})
+@Retention(RetentionPolicy.RUNTIME)
+public @interface DoAuth {
+    boolean required() default true;
+}
```
Comment (jovany-wang) [inline src/main/java/com/cloud/common/annotations/DoAuth.java]: One more line

File: src/main/java/com/cloud/common/annotations/NoAuth.java [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target({ElementType.METHOD, ElementType.TYPE})
+@Retention(RetentionPolicy.RUNTIME)
+public @interface NoAuth {
+    boolean required() default true;
+}
```
Comment (jovany-wang) [inline src/main/java/com/cloud/common/annotations/NoAuth.java]: the same as here.

File: src/main/java/com/cloud/common/config/WebMvcConfig.java [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.stereotype.Component;
+import org.springframework.web.servlet.HandlerInterceptor;
+import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
+import org.springframework.web.servlet.config.annotation.ResourceHandlerRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+/**
+ * Allow access to static resources
```
Comment (jovany-wang) [inline src/main/java/com/cloud/common/config/WebMvcConfig.java]:
```suggestion
 * Allow access to static resources.
```

File: script/dkv_run.sh [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+  wget https://distkv-1252912764.cos.ap-shanghai.myqcloud.com/distkv-server-jars/nightly/distkv-server.jar
+fi
+
+cp ../target/distkv-cloud-0.1.0-SNAPSHOT-jar-with-dependencies.jar ./distkv_cloud.jar
+cp ../src/main/java/com/distkv/tool/dashboard/index.html ./index.html
+
+nohup java -cp ./distkv-server.jar com.distkv.server.storeserver.StoreServer >> /dev/null 2>&1 &
+
+sleep 2
+nohup java -cp ./distkv_cloud.jar com.distkv.tool.dashboard.DashboardServer >> /dev/null 2>&1 &
```
Comment (jovany-wang) [inline script/dkv_run.sh]: ???

File: src/main/resources/users.json [hunk 7]
```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "username": "jovany-wang",
```
Comment (jovany-wang) [inline src/main/resources/users.json]: Why put this into repo?

**Review by jovany-wang — APPROVED** [review]

**Pull request merged**